processing/src/main/java/io/druid/segment/StringDimensionIndexer.java

```diff
@@ -265,6 +265,20 @@ public StringDimensionIndexer(MultiValueHandling multiValueHandling, boolean has
     return encodedDimensionValues;
   }
 
+  @Override
+  public long estimateEncodedKeyComponentSize(int[] key)
+  {
+    /**
```

Comment:
The javadoc-style `/**` comment isn't appropriate here (those are meant for documenting classes, methods, and fields). It could just be a `//` style comment since it's one line. And usually we use that style within methods even for multiline comments (although sometimes we use `/*`).

Reply:
ok